packages/block-editor/src/components/use-setting/index.js

```diff
@@ -122,7 +123,18 @@ export default function useSetting( path ) {
 				return undefined;
 			}
 
-			let result;
+			// 0. Allow third-parties to filter into the block's settings at runtime.
```

Comment:
```suggestion
			// 0. Allow third parties to filter the block's settings at runtime.
```
Nit: Slight wording tweak.